cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc

```diff
@@ -12,13 +12,19 @@ If you import a VMware template, the wizard creates a virtual machine based on t
 include::modules/technology-preview.adoc[leveloffset=+1]
 :!FeatureName:
 
-.Prerequisites
+include::modules/cnv-creating-vddk-image.adoc[leveloffset=+1]
 
-* If you use NFS-backed storage for the 2 GB disk that is attached to the Conversion Pod, you must link:https://access.redhat.com/solutions/4982751[configure a hostPath volume].
+ifeval::["{CNVVersion}" == "2.3"]
+[NOTE]
+====
+If you use NFS-backed storage for the 2 GB disk that is attached to the Conversion Pod, you must configure a hostPath volume. See xref:cnv-importing-vmware-vm.adoc#known-issues_cnv-importing-vmware-vm[].
+====
+endif::[]
 
-include::modules/cnv-creating-vddk-image.adoc[leveloffset=+1]
 include::modules/cnv-importing-vmware-vm.adoc[leveloffset=+1]
 
+../cnv/cnv_virtual_machines/cnv_importing_vms/modules/cnv-troubleshooting-vmware-vm-import.adoc
```

Comment:
This link's incomplete.

Reply:
No, I pasted it there by accident! Thanks. Got rid of it. It's been a long week.... :-)